### PR TITLE
Add preliminary notes for 1.8.4

### PR DESCRIPTION
**Pull request by miekg**
Draft notes for 1.8.4

Signed-off-by: Miek Gieben <miek@miek.nl>

### CODE REVIEW

File: notes/coredns-1.8.4.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Chris O'Haver,
+Miek Gieben,
+Mohammad Yosefpor,
+Paco Xu,
+Soumya Ghosh Dastidar.
+
+## Noteworthy Changes
+
+* plugin/metrics: remove RR type (https://github.com/coredns/coredns/pull/4534)
```
Comment (SuperQ) [inline notes/coredns-1.8.4.md:25]: Maybe be more specific about removal from histograms.

File: notes/coredns-1.8.4.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* plugin/health: add logging for local health request (https://github.com/coredns/coredns/pull/4533)
+* plugin/bind: Bind by interface name (https://github.com/coredns/coredns/pull/4522)
+* plugin/forward: Add upstream metadata (https://github.com/coredns/coredns/pull/4521)
+* plugin/minimal: Add minimal-responses plugin (https://github.com/coredns/coredns/pull/4417)
+* plugin/transfer: reply with refused (https://github.com/coredns/coredns/pull/4510)
+* plugin/kubernetes: TODO endpoint slices
```
Comment (chrisohaver) [inline notes/coredns-1.8.4.md]:
```suggestion
* plugin/kubernetes: do endpoint/slice check in retry loop (https://github.com/coredns/coredns/pull/4492)
```

File: notes/coredns-1.8.4.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Soumya Ghosh Dastidar.
+
+## Noteworthy Changes
+
+* plugin/metrics: remove RR type (https://github.com/coredns/coredns/pull/4534)
+* plugin/health: add logging for local health request (https://github.com/coredns/coredns/pull/4533)
+* plugin/bind: Bind by interface name (https://github.com/coredns/coredns/pull/4522)
+* plugin/forward: Add upstream metadata (https://github.com/coredns/coredns/pull/4521)
+* plugin/minimal: Add minimal-responses plugin (https://github.com/coredns/coredns/pull/4417)
+* plugin/transfer: reply with refused (https://github.com/coredns/coredns/pull/4510)
```
Comment (chrisohaver) [inline notes/coredns-1.8.4.md:31]:
```suggestion
* plugin/transfer: reply with refused (https://github.com/coredns/coredns/pull/4510)
* plugin/kubernetes: Exclude unready endpoints from endpointslices (https://github.com/coredns/coredns/pull/4580)
```

File: notes/coredns-1.8.4.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Miek Gieben,
+Mohammad Yosefpor,
+Paco Xu,
+Soumya Ghosh Dastidar.
+
+## Noteworthy Changes
+
+* plugin/metrics: remove RR type (https://github.com/coredns/coredns/pull/4534)
+* plugin/health: add logging for local health request (https://github.com/coredns/coredns/pull/4533)
+* plugin/bind: Bind by interface name (https://github.com/coredns/coredns/pull/4522)
```
Comment (chrisohaver) [inline notes/coredns-1.8.4.md:27]:
```suggestion
* plugin/bind: Bind by interface name (https://github.com/coredns/coredns/pull/4522)
* plugin/bind: Exclude interface or ip address  (https://github.com/coredns/coredns/pull/4543)
```

File: notes/coredns-1.8.4.md (hunk 1 above)
Comment (chrisohaver) [reply notes/coredns-1.8.4.md:25]: Should we bump minor version for changes to existing metrics?

**Pull request merged**